### DISCUSSION TITLE
Unable to use all open_iconic_play_2x2 icons

### DISCUSSION THREAD

**Discussion by m-r-m-s** [olikraus/u8g2#1540] [Q&A] [answered]
Hi @olikraus and others,

I am trying out the [open_iconic_play_2x2 icons](https://github.com/olikraus/u8g2/wiki/fntgrpiconic#open_iconic_play_2x2) in the u8x8 "HelloWorld" example .ino and
I do not have access to display the all of the listed icons in the 64/40 range:

![Screenshot 2021-08-02 at 15-26-10 fntgrpiconic · olikraus u8g2 Wiki](https://user-images.githubusercontent.com/16727232/127914192-1c6c0e44-52a4-4bfd-bf64-2ad24308ab8f.png)

The last icon I am able to display using the drawUTF8 function is`u8x8.drawUTF8(0,0,"\u0049");` and then it rolls over to the final two icons (80/50).

What am I missing here? Any help is greatly appreciated. Thank you.

**Comment by olikraus** [comment, marked as answer]
Did you try u8x8_DrawGlyph? Something like
``` C++
for( int i = 64; i < 82; i++ )
   u8x8.drawGlyph( (i-64) & 7, (i-64) / 8, i);
```
should draw all the gylphs.

**Reply by m-r-m-s** [reply]
Thank you it does display all of them. To clarify: how do I display them individually? I assumed that using the drawUTF8 function I could display each one individually, as a single icon with that function e.g. `u8x8.drawUTF8(0,0, "\u0049");`
But I cannot do this with the ones highlighted in red.

**Reply by olikraus** [reply]
There is no need to use unicode here, better use drawString, something like:
`u8x8.drawString(0,0,"\x40\x41\x42"`, see also https://en.wikipedia.org/wiki/Escape_sequences_in_C

**Reply by olikraus** [reply]
Actually you could use `8x8.drawString(0,0,"@abc")`.

**Reply by m-r-m-s** [reply]
> Actually you could use `8x8.drawString(0,0,"@abc")`. 

Hmmmn, I am not sure I understand this last example? What is `@abc` ? I see it displays the first icon but I still cannot figure out how to access the above icons in red individually. For example: How do I only display this one icon here:
![Screenshot 2021-08-02 at 15-26-10 fntgrpiconic · olikraus u8g2 Wiki](https://user-images.githubusercontent.com/16727232/128036209-028061c1-c070-48ef-8cee-431bc50aebc7.png)

Also, I tried the escape sequence format using the drawString function but I still run into the same issue: what are the escape sequences for the individual icons highlighted in red? My apologies if I am not asking the question correctly, I am a beginner and still learning the library. Thank you again for your time.

**Reply by v1993** [reply]
They should be `\x4A` to `\x4F` if I got it right. 64/40 is first character's code in decimal/hex, so eleventh character is 4A.

**Reply by olikraus** [reply]
Correct

**Reply by olikraus** [reply]
The requested glyph can be printed also with "k" if I counted correctly.

**Reply by m-r-m-s** [reply]
> They should be `\x4A` to `\x4F` if I got it right. 64/40 is first character's code in decimal/hex, so eleventh character is 4A.

Thank you. This is starting to make sense now. Thanks so much for your patience =D

**Reply by m-r-m-s** [reply]
> The requested glyph can be printed also with "k" if I counted correctly.

Thank you this works for me `u8x8.drawGlyph(0,2, 'K');` Many thanks for supporting me through these small steps =D

**Reply by olikraus** [reply]
Ah, yes, uppercase 🤪
Instead of `@abc` the sequence `@ABC` will print the first 4 icons.
